conf/eslint.json

```diff
@@ -98,6 +98,7 @@
         "no-undef": 2,
         "no-undef-init": 2,
         "no-undefined": 0,
+        "no-unexpected-multiline": 2,
         "no-underscore-dangle": 2,
```

Comment:
Is this because you think it's not useful enough to be on by default, or because eslint is moving to all rules off-by-default?